mydumper-overview.md

```diff
@@ -84,6 +84,10 @@ The following TiDB versions supports the `_tidb_rowid` index:
 
 ## FAQ
 
+### How to resolve the error that occurs when the `--tidb-snapshot` option is used to export data?
+
+In this situation, you need to add a `--skip-tz-utc` option. Otherwise, Mydumper will pre-configure the UTC time zone and convert the time zone when the `tidb-snapshot` is configured, which causes this error.
```

Comment:
```suggestion
In this situation, you need to add a `--skip-tz-utc` option. Otherwise, Mydumper will pre-configure the UTC time zone and convert the time zone when `tidb-snapshot` is configured, which causes this error.
```